tests/test_diff_tally/test_diff_tally.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+
+import glob
+import os
+try:
+    from StringIO import StringIO
+except:
+    from io import StringIO
```

Comment:
Looks like this is needed because of some string vs unicode issue.  In Python2, the `from io` version will fail when `df.to_csv` passes it a string.